src/parser/mumexmlparser.cpp

```diff
@@ -389,12 +393,13 @@ QByteArray MumeXmlParser::characters(QByteArray& ch)
       toUser.append(ch);
       break;
 
-      case XML_DESCRIPTION: // static line
-        removeAnsiMarks(m_stringBuffer) ; //remove color marks
-        m_staticRoomDesc += m_stringBuffer+"\n";
-        if (m_examine || !Config().m_brief)
+  case XML_DESCRIPTION: // static line
+      removeAnsiMarks(m_stringBuffer) ; //remove color marks
+      m_staticRoomDesc += m_stringBuffer+"\n";
+      if (!m_gratuitous) {
```

Comment:
I'm wondering if I should leave in `m_brief` support since I didn't remove it... meh?